commands/build.py

```diff
@@ -57,8 +58,11 @@ def prepare(self):
     def make(self):
         cmd = ["cmake",
                "--build", self.build_path,
-               "--target", "install",
-               "-j" if not self.make_verbose else "-v"]
+               "--target", "install"]
+        if self.make_verbose:
```

Comment:
make_verbose flag is broken (due to the --target change). The following works with ninja:
```
if self.make_verbose:
    cmd.extend(["--", "-v"])    
else:
    cmd.extend(["--", "-j", str(multiprocessing.cpu_count() + 1)])
```

But fails for make, no -v there - need to use VERBOSE=1 instead.

Reply:
Grmbl... `-v` was introduced in CMake 3.14, apparently...

I'll just convert it to VERBOSE=1 which is only supported in Make. If you need verbose build, you need to use make. Sounds fair.